pg_lakehouse/tests/explain.rs

```diff
@@ -26,6 +26,7 @@ const S3_BUCKET: &str = "test-trip-setup";
 const S3_KEY: &str = "test_trip_setup.parquet";
 
 #[rstest]
+#[ignore = "EXPLAIN not fully working"]
```

Comment:
could you be more specific about 'not fully working'? Is it a particular error?

Reply:
I just created an issue for it here: https://github.com/paradedb/paradedb/issues/1298